Tests/SwiftDocCUtilitiesTests/ArgumentParsing/ConvertSubcommandTests.swift

```diff
@@ -25,7 +25,41 @@ class ConvertSubcommandTests: XCTestCase {
         // test console output with unrelated messages.
         Docc.Convert._errorLogHandle = .none
     }
-    
+
+    override func setUpWithError() throws {
+        // By default, run all tests in a temporary directory to ensure that they are not affected
+        // by the machine environment.
+        let priorWorkingDirectory = FileManager.default.currentDirectoryPath
+        let temporaryDirectory = try createTemporaryDirectory()
+        FileManager.default.changeCurrentDirectoryPath(temporaryDirectory.path())
+        addTeardownBlock {
+            FileManager.default.changeCurrentDirectoryPath(priorWorkingDirectory)
+        }
```

Comment:
Would it make sense to set the default template here as well? It seems to me like all tests except `testOptionsValidation` and `testTransformForStaticHostingFlagWithoutHTMLTemplate` want the default template to be set but that it's easy forget. 

For example, are these tests expected to set the default template or not? 
- `testEmitLMDBIndex`
- `testExperimentalEnableCustomTemplatesFlag`
- `testExperimentalEnableDeviceFrameSupportFlag`
- `testExperimentalEnableExternalLinkSupportFlag`
- `testExperimentalEnableOverloadedSymbolPresentation`
- `testParameterValidationFeatureFlag`

```suggestion
        }
        setTemplateEnvironmentVariable(testTemplateURL.path)
```

Reply:
That's probably the best. I think i wasn't sure how to manage the fact that `testOptionsValidation` sets the default template during the test, but i think setting it to a reasonable default and letting it mess with it later isn't a problem. I'll make that change.